addons/l10n_hk/tests/test_emv_qr.py

```diff
@@ -0,0 +1,89 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.exceptions import UserError
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestEMVQRCode(AccountTestInvoicingCommon):
```

Comment:
There is another class named `TestEMVQRCode`, perhaps it would be better to prepend the class name with `L10nHk` (same with the functions) ?